Documentation/General/finalReport/reflection.tex

```diff
@@ -77,6 +77,28 @@ \section{SCRUM}
 Overall, we're all convinced SCRUM is a great way to develop software, however we think that a sprint could better last 2 or 3 weeks instead of one.
 The sprint planning and sprint reflection took up a considerable amount of time in the sprint and the weekly demo forced us to put a difficult issue aside more than once.
 
+\section{Pull-based development and code reviews}
+While developing MoodCat, most of us had our first real encounter with pull-based development.
+This is a technique where you use your versioning system (in our case Git) and it's capability to create branches to its full potential. 
+The basic structure here is:
+\begin{itemize}
+\item The Product Owner picks tickets from the product backlog and puts them on the sprint backlog.
```

Comment:
In our project is was the whole group not the PO that decided who does what.

Maybe: On the weekly sprint meeting tickets from the product backlog where selected and put on the sprint backlog.